packages/tools/tests/test/playwright/visualizationPlaywright.utils.ts

```diff
@@ -98,6 +98,7 @@ export const evaluatePlaywrightVisTests = async (engineType = "webgl2", testFile
             continue;
         }
         test(testCase.title, async ({ page }) => {
+            console.log("Running test: " + testCase.title, "Meta: ", testCase.playgroundId || testCase.scriptToRun || testCase.sceneFilename);

```

Comment:
Just to be sure, as several tests can run in parallel, we are sure that this log can't appear before some other logs that would pertain to another test (because a test would take more time to execute than another one, for eg)?